docs/irf/psf.rst

```diff
@@ -1,10 +1,42 @@
 .. _irf-psf:
 
-Point Spread Function
-=====================
+PSF
+===
+as a function of of true energy and offset angle (:ref:`gadf:psf_table`)
+------------------------------------------------------------------------
+The `~gammapy.irf.PSF3D` class represents the spatial probability of an event with true position :math:`p_{\rm true}` to
+have an estimated position :math:`p_{\rm reco}`, as a function of true energy and offset angle from the field of view center
+(:math:`PSF(p_{\rm reco}|p, E)` in :ref:`irf-theory`).
 
-TODO: describe / illustrate the existing classes:
+Its format specifications are available in :ref:`gadf:psf_table`.
 
-* `~gammapy.irf.PSFKernel`
-* `~gammapy.irf.PSF3D`
+This is the format in which IACT DL3 PSFs are usually provided, as an example:
 
+.. plot:: irf/plot_psf.py
+    :include-source:
+
+
+as a function of true energy (:ref:`gadf:psf_gtpsf`)
+----------------------------------------------------
+`~gammapy.irf.EnergyDependentTablePSF` instead represents an energy dispersion as a function of true energy only 
+(:math:`PSF(p_{\rm reco}| E)` following the notation in :ref:`irf-theory`). 
+
+Its format specifications are available in :ref:`gadf:psf_gtpsf`.
+
+Such a PSF can be obtained by evaluating the `~gammapy.irf.PSF3D` at a given offset (using `~astropy.coordinates.Angle`)
+
+.. plot:: irf/plot_psf_table.py
+    :include-source:
+
+**Note:** the offset angle on the x axis is not an offset angle from the field of view center but the distance between 
+the source position and the reconstructed gamma ray position.  
```

Comment:
For this we have introduced the convention to name it "rad", could be mentioned here...